src/main/java/com/greensopinion/gradle/android/eclipse/GenerateLibraryDependenciesAction.java

```diff
@@ -65,22 +65,23 @@ public void execute(Classpath classpath) {
 
 	private Stream<ClasspathEntry> explodeAarJarFiles(Library aarLibrary) {
 		File aarFile = new File(aarLibrary.getPath());
-		File targetFolder = new File(new File(new File(project.getProjectDir(), "build"), "exploded-aars"),
-				aarFile.getName());
+    String jarId = aarLibrary.getModuleVersion().toString().replaceAll(":", "-");
```

Comment:
Looks like there's a spaces-vs-tabs formatting issue, could you reformat with the same formatting (was tabs?)

Same here and below.